mantidproject/mantid
# Update matplotlib to 3.5 in conda

**Description of work.**
A lot of small patches to ensure matplotlib 3.5.* compatibility with workbench



**To test:**
- Using your mantid conda developer environment, build, and launch workbench. Ensure you have matplotlib 3.5.1 or higher.
- Load a workspace capable of 3D plots
- Open two, 3D plots surface and wireframe
- Save your project
- Load project
- 2 new plots should have opened the same as the two originals
- No warnings, errors, or exceptions should have occured.



Fixes #33529 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
